### ISSUE TITLE
[Errors] with blocks and global vars

### ISSUE THREAD

**Issue by mhegazy** [microsoft/TypeScript#47]
``` TypeScript
with (window) {
        var p = () => this;
}
```

Expected: no error
Actual: 

``` TypeScript
with (window) {
        !!! Could not find symbol 'window'.
        var p = () => this;
    }
```
